docs/news.md

```diff
@@ -1,4 +1,19 @@
 # News and Announcements
+## 2019
+### 5th April
+We're delighted to announce the release of v1.0 of the SAFE Stack! It's been an exciting year or so for the stack as the stack has evolved and matured. We now have what we feel is a stable and feature-rich stack that achieves the original goals of SAFE, plus a number of optional libraries that people can take advantage of.
+
+With the release of v1, you can now do the following:
+
+* Create a fully working, full-stack web application in F# using the SAFE template in just a few seconds and have it running on your machine from clean in less than a minute.
+* Choose flexible hosting options, including Azure App Service, Google Cloud Platform and Docker.
+* Host with multiple server models including Saturn and Giraffe.
+* Have simultaneous client and server-side debugging sessions in VS Code.
+
+You can upgrade the SAFE Stack template via the dotnet CLI: `dotnet new -i SAFE.Template` to get the latest and greatest version, or `dotnet new -i SAFE.Template::1.0` to pin to v1.0.
```

Comment:
I didn't know about pinning - maybe we should emphasise it more e.g. by placing the information in quick-start?

Reply:
Can also do that - happy to do if you like.

Reply:
Not sure if that's appropriate for the quick start though?

Reply:
Maybe not quick start, but elsewhere so that it's discoverable